heartbeat-app/src/main/kotlin/edu/stanford/bdh/heartbeat/app/fake/FakeConfigs.kt

```diff
@@ -0,0 +1,9 @@
+package edu.stanford.bdh.heartbeat.app.fake
+
+object FakeConfigs {
```

Comment:
@pauljohanneskraft you simulate a fake flow via this object. If enabled it runs a survey from data of a json file that contains at least one question of the field types that we should implement